features/braille/brailleSlice.js

```diff
@@ -69,18 +76,26 @@ export const brailleSlice = createSlice({
       }
       return newState
     },
+    inputItemClick: (state, action) => {
+      const { itemIdx } = action.payload
+      state.cursorIdx = itemIdx
+      state.cursorType =
+        state.cursorIdx === state.confirmedInput.length &&
+        state.selected.length === 0
```

Comment:
Remove `state.selected.length === 0` or remove the whole inputItemClick, as it is not fully functional now